overlay.nix

```diff
@@ -191,6 +191,11 @@ let
 
         pysisyphus = super.python3.pkgs.toPythonApplication self.python3.pkgs.pysisyphus;
 
+        q-chem-installer = callPackage ./pkgs/apps/q-chem/installer.nix { };
+        q-chem = callPackage ./pkgs/apps/q-chem/default.nix {
```

Comment:
Please put `q-chem` after the molpro section and make it conditional (e.g. `// lib.optionalAttrs (cfg.licQChem != null)`). That will avoid evaluation errors, when there is no license present.